apps/masterbots.ai/components/shared/wordware-chat.tsx

```
@@ -0,0 +1,247 @@
+        const response = await fetch(
+          `/api/wordware/describe?promptId=${promptId}`
+        )
+        const data = await response.json()
+        if (!response.ok) {
+          throw new Error(data.error || 'Failed to fetch prompt details')
+        }
+        setPromptDetails(data)
+        setInputs(
+          data.inputs.reduce(
+            (acc: any, input: { label: any }) => ({
+              ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)